dynamic_dynamodb/config/config_file_parser.py

```diff
@@ -263,7 +263,13 @@
     	'option': 'rotate_scavenge',
     	'required': False,
     	'type': 'int'
-    },    
+    },   
+    {
```

Comment:
Can you change this to be rotate_interval_unit (month)?
Add rotate_window_start to be the minutes in advance before creating the next time series table